Dockerfile

```diff
@@ -18,7 +18,10 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
+# !! The following scrips are needed to correctly execute migrations not only on local build but from DockerHub image as well !! 
+# Copy the entrypoint script
+COPY entrypoint.sh /usr/local/bin/entrypoint.sh
+RUN chmod +x /usr/local/bin/entrypoint.sh
```

Comment:
No need to create the script for running migration. You can leave it as is, or add a migration command, and ENTRYPOINT

`ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]`

Reply:
> No need to create the script for running migration. You can leave it as is, or add a migration command, and ENTRYPOINT
> 
> `ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]`

thanks, you saved me few hours of debugging